RA Scripts/Legend of Zelda, The The Minish Cap.rascript

```diff
@@ -600,22 +600,23 @@ function MusicId() => byte(0x029EF4)
 
 function CreditsLoaded() => MusicId() == 0x09 && Delta(MusicId()) == 0x00
 
-function ScreenIdByte() => byte(0x000BF4)
-function ScreenIdWord() => word(0x000BF4)
+function ScreenId() => word_be(0x000BF4)
```

Comment:
For memory accessor wrappers, we want to use camelCase. For my scripts, I reserve PascalCase for computables and other functions.

Reply:
Though I do acknowledge that the original was also in PascalCase. This was an error on my part.